.changeset/big-gifts-notice.md

```diff
@@ -0,0 +1,6 @@
+---
+'layerchart': patch
+---
+
+Fixed `strokeWidth` prop on `Bar` component.
+Made `Spline` pass `{...$$restProps}` to its `<path>` element.
```

Comment:
It's usually best to create 2 separate changeset files for each of these so they are separate CHANGELOG items.  I can hand edit the CHANGELOG before merging the release PR.  Thanks again!